dist/bin/kctf-cluster

```diff
@@ -320,11 +320,98 @@ EOF
   echo "${CONFIG_NAME}"
 }
 
+function kctf_cluster_get_ip_ranges {
+  get_cloud_armor_policy || return
+  _kctf_log "Current IP range: \"$ret\""
+}
+
+function kctf_cluster_ip_ranges_usage {
+  echo "usage: kctf cluster set-src-ip-ranges ranges" >&2
+  echo "  -h|--help       print this help" >&2
+  echo "  ranges          ip ranges to allow of the form 1.1.1.1/32,2.2.2.0/24" >&2
+}
+
+function kctf_cluster_ip_ranges {
+  OPTS="h"
+  LONGOPTS="help"
+  PARSED=$(getopt --options=$OPTS --longoptions=$LONGOPTS --name "kctf cluster set-src-ip-ranges" -- "$@")
+  if [[ $? -ne 0 ]]; then
+    kctf_cluster_ip_ranges_usage
+    return 1
+  fi
+  eval set -- "$PARSED"
+
+  while true; do
+    case "$1" in
+      -h|--help)
+        kctf_cluster_ip_ranges_usage
+        return 1
+        ;;
+      --)
+        shift
+        break
+        ;;
+      *)
+        _kctf_log_err "Unrecognized argument \"$1\"."
+        kctf_cluster_ip_ranges_usage
+        return 1
+        ;;
+    esac
+  done
+
+  if [[ $# -ne 1 ]]; then
+    _kctf_log_err "kctf cluster set-src-ip-ranges: expecting a single range argument"
+    kctf_cluster_ip_ranges_usage
+    return 1
+  fi
+
+  RANGES="$1"
+
+  # update the cloud armor policy
+  set_cloud_armor_policy "${RANGES}" || return
+
+  # stop the operator
+  "${KCTF_BIN}/kubectl" delete -f "${KCTF_CTF_DIR}/kctf/resources/operator.yaml" || return
+
+  start_operator_gce || return
```

Comment:
what happens when someone runs this on a local cluster?

Reply:
added a check for type == gce